### ISSUE TITLE
Save metadata across restarts in addition to console contents

### ISSUE THREAD

**Issue by lmcmicu** [ontodev/droid#27]
Issue https://github.com/ontodev/droid/issues/26 was to persist the console data across server restarts. We should save process metadata as well: start time, who launched it, command name, etc.

**Comment by lmcmicu** [comment]
This issue is ready to be closed, but the branch that closes it is based on the [cgi-scripts](https://github.com/ontodev/droid/tree/cgi-scripts) branch, so we should wait until https://github.com/ontodev/droid/pull/36 has been merged before creating a PR to close this issue.

**Comment by lmcmicu** [comment]
Note that the "who launched it" information is not saved. I don't think this was something that was originally needed for this feature, though, but instead just something I added to the description (by mistake) when I created the issue.

The "who launched it" information is _logged_ once a process is started, but this information is not actually saved anywhere. My understanding is that we don't really need to save this info, but I should be corrected if I am wrong (cc @jamesaoverton )

What is saved currently is enough info so that the branch page will display as much as possible of what it displayed before the server restart (command, action, status, console, etc.).
